v2/contribute/decisions/thirdparty/0007-check-for-enabled-recipes.mdx

```diff
@@ -35,12 +35,15 @@ Chosen option: **Recipes should check**, because
 
 ### Recipes should not check
 
-Recipes functions/APIs don't check for this flag, and, all the statically configured recipes will function as defined, even if the user has disabled them in the core.
```

Comment:
make it very clear that recipe functions don't check and only APIs check and write down why we decided this.